scripts/dist-tags-promote

```diff
@@ -1,5 +1,14 @@
+# get CI environment auth'd; local one-off use assumes you're already auth'd to npm
+if [ $NPM_TOKEN ]; then
+  echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc
+fi
+
 CLI_RC_VERSION=$(npm show sfdx-cli --json | jq -r '.["dist-tags"] | .["latest-rc"]')
```

Comment:
If you want, you can just do `npm show sfdx-cli@latest-rc version`

Reply:
yeah, that's so much better.